lib/mosaic-database/src/main/java/org/eclipse/mosaic/lib/database/persistence/TABLES.java

```diff
@@ -16,31 +16,28 @@
 package org.eclipse.mosaic.lib.database.persistence;
 
 final class TABLES {
-    static String PROPERTIES = "Properties";
-    static String NODE = "Node";
-    static String WAY = "Way";
-    static String WAY_CONSISTS_OF = "WayConsistsOf";
-    static String CONNECTION = "Connection";
-    static String CONNECTION_CONSISTS_OF = "ConnectionConsistsOf";
-    static String RESTRICTION = "Restriction";
-    static String TRAFFIC_SIGNALS = "TrafficSignals";
-    static String ROUTE = "Route";
-    static String ROUNDABOUT = "Roundabout";
-    static String ROUNDABOUT_CONSISTS_OF = "RoundaboutConsistsOf";
-    static String BUILDING = "Building";
-    static String BUILDING_CONSISTS_OF = "BuildingConsistsOf";
-    static String CONNECTION_DETAILS = "ConnectionDetails";
+    final static String PROPERTIES = "Properties";
+    final static String NODE = "Node";
+    final static String WAY = "Way";
+    final static String WAY_CONSISTS_OF = "WayConsistsOf";
+    final static String CONNECTION = "Connection";
+    final static String CONNECTION_CONSISTS_OF = "ConnectionConsistsOf";
+    final static String RESTRICTION = "Restriction";
+    final static String TRAFFIC_SIGNALS = "TrafficSignals";
+    final static String ROUTE = "Route";
+    final static String ROUNDABOUT = "Roundabout";
+    final static String ROUNDABOUT_CONSISTS_OF = "RoundaboutConsistsOf";
+    final static String BUILDING = "Building";
+    final static String BUILDING_CONSISTS_OF = "BuildingConsistsOf";
+    final static String CONNECTION_DETAILS = "ConnectionDetails";
     /**
      * Will no longer be written to, but used to update old versions of databases.
      */
     @Deprecated
-    static String CORNER = "Corner";
+    final static String CORNER = "Corner";
     /**
      * Will no longer be written to, but used to update old versions of databases.
      */
     @Deprecated
     static String WALL = "Wall";
```

Comment:
Just for unity, could you set WALL to `final`?
(otherwise, everything is fine)

Reply:
good catch! I changed it accordingly